packages/flutter_tools/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   archive: ^1.0.20
   args: ^0.13.4
+  collection: '>=1.9.1 <2.0.0'
```

Comment:
Is there any way to get out of having this dependency? We've been trying to drop our dependency on package:collection...